pkg/bootstrap/docker/openshift/login.go

```diff
@@ -32,6 +33,32 @@ func Login(username, password, server, configDir string, f *clientcmd.Factory, c
 	for k := range adminConfig.AuthInfos {
 		adminConfig.AuthInfos[k].LocationOfOrigin = ""
 	}
+	serverFound := false
+	for k := range adminConfig.Clusters {
+		adminConfig.Clusters[k].LocationOfOrigin = ""
+		if adminConfig.Clusters[k].Server == server {
+			serverFound = true
```

Comment:
We do want to assign it to all clusters. We want to remove the original LocationOfOrigin which points to a path inside the container.